### PR TITLE
Add file to owned templates

### PR DESCRIPTION
**Pull request by Arthurlpgc**

### CODE REVIEW

File: vitaes-webapp/src/App.js [hunk 1]
```diff
@@ -64,6 +64,28 @@ class App extends Component {
     }).catch(function(error) {
       app.setState({tab: 0})
     })
+
+    this.hostname = window.location.hostname + ':5000'
```
Comment (Arthurlpgc) [inline vitaes-webapp/src/App.js]: Will refactor into a function

File: vitaes-webapp/src/AddTemplate.js [hunk 2]
```diff
@@ -22,12 +23,58 @@ class TemplateField extends Component {
   }
 }
 
+class OwnedTemplate extends Component {
```
Comment (RamonSaboya) [inline vitaes-webapp/src/AddTemplate.js]: Wouldn't it be better to declare functions out of the JSX?

File: vitaes-webapp/src/AddTemplate.js [hunk 3]
```diff
@@ -46,7 +93,7 @@ class AddTemplate extends Component {
                   'Content-Type': 'application/json',
                 },
                 body: JSON.stringify(this.state.template)}); 
-              this.setState({template: {owner: firebase.auth().currentUser.uid, command: "", likes: 0, name: "", params: {}, fixed_params:{}}})}}
+              this.setState({template: {owner: firebase.auth().currentUser.uid, command: "", data: {likes: 0}, name: "", params: {}, fixed_params:{}}})}}
```
Comment (RamonSaboya) [inline vitaes-webapp/src/AddTemplate.js]: Maybe create a const with default state?

Comment (Arthurlpgc) [reply vitaes-webapp/src/AddTemplate.js]: Unsure on how to proceed with JS, as it is all by reference, will check on it.

**Pull request merged**